views/core/member/viewAll.hbs

```diff
@@ -13,9 +12,7 @@
     <section class="content">
       <div class="row">
         <div class="col-md-3">
-          <a href="/core/member/create" style="color: white;">
-            <button type="button" class="btn btn-block btn-success btn-lg box">Nouveau membre</button>
-          </a>
+          <a href="/core/member/signin" style="color: white;"><button type="button" class="btn btn-block btn-success btn-lg box">Nouveau membre</button></a>
```

Comment:
Je préférais le "create" d'avant, pas "signin". Premièrement , sign in = se connected. Toi tu veux dire "sign up" je pense, qui veut dire "inscription".

Cependant pour moi c'est moi une incription et plus une création puisque ce n'est pas le membre en question qui le fait mais le SG